R/read10xVisium.R

```diff
@@ -125,7 +125,7 @@ read10xVisium <- function(samples="",
     
     dir <- file.path(samples, "spatial")
     xyz <- file.path(
-        rep(dir, each = length(sids)), 
+        rep(dir, each = 2), 
```

Comment:
Could we perhaps assign the character vector in line 131 to a variable `sfx` (or something, for the file name suffix), and used `length(sfx)` here? I feel like the `2` isn't very explicit or robust...

Reply:
I agree with @HelenaLC's comment here, thanks for noticing that!